pkgs/development/python-modules/unicode-slugify/default.nix

```diff
@@ -18,16 +18,20 @@ buildPythonPackage rec {
     sha256 = "25f424258317e4cb41093e2953374b3af1f23097297664731cdb3ae46f6bd6c3";
   };
 
+  patches = [ ./use_pytest_instead_of_nose.patch ];
```

Comment:
This should ideally be sent upstream. We may also want to include https://github.com/mozilla/unicode-slugify/pull/41.